src/Html2Text.php

```diff
@@ -23,8 +24,8 @@ public static function convert(string $html, Config $config = new Config()): str
         }
 
         $html = static::fixNewlines($html);
-        if (mb_detect_encoding($html, "UTF-8", true)) {
-            $html = mb_convert_encoding($html, "HTML-ENTITIES", "UTF-8");
+        if (mb_detect_encoding($html, 'UTF-8', true)) {
+            $html = Mbstring::mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8');
```

Comment:
Used this to avoid requiring a mapping for entity replacement, and avoiding the deprecation warning in PHP 8.2. 👍🏻 This probably isn't a permanent fix though...